### PR TITLE
Add Arista Networks to snmp.yml and generator.yml

### PR DESCRIPTION
**Pull request by ninoabbate**
Arista Networks MIBs can be found here: https://www.arista.com/en/support/product-documentation/arista-snmp-mibs. Tested on Arista DCS-7010T-48 switch.

### CODE REVIEW

**Comment by SuperQ** [comment]
You may want to include `ifXTable` in the walk so that you get the "High Capacity" metrics.

File: generator/generator.yml [hunk 1]
```diff
@@ -122,3 +122,21 @@ modules:
       - 1.3.6.1.4.1.25461.2.1.2.3 # panSession
       - 1.3.6.1.4.1.25461.2.1.2.5 # panGlobalProtect
 
+# Arista Networks
+#
+# Arista Networks MIBs can be found here: https://www.arista.com/en/support/product-documentation/arista-snmp-mibs
+#
+# https://www.arista.com/assets/data/docs/MIBS/ARISTA-ENTITY-SENSOR-MIB.txt
```
Comment (brian-brazil) [inline generator/generator.yml]: These aren't the ones listed in the readme

File: generator/generator.yml [hunk 2]
```diff
@@ -122,3 +122,21 @@ modules:
...
+# https://www.arista.com/assets/data/docs/MIBS/ARISTA-SW-IP-FORWARDING-MIB.txt
+#
+# Tested on Arista DCS-7010T-48 switch
+#
+  arista_sw:
+    walk: 
+      - sysUpTime
+      - interfaces
```
Comment (brian-brazil) [inline generator/generator.yml]: Are there any usable unique lookups on ifName/ifDescr/ifAlias we can use here?

File: generator/generator.yml (hunk 1 above)
Comment (ninoabbate) [inline generator/generator.yml]: Do I need to change the link in readme with this?
https://www.arista.com/en/support/product-documentation/arista-snmp-mibs

Comment (brian-brazil) [inline generator/generator.yml]: Please list the exact files that need to be downloaded

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**